crates/primitives/contract/src/class.rs

```diff
@@ -232,18 +230,18 @@ pub mod convert {
                 r#type: AbiStructTypeWrapper::from(abi_struct_entry.r#type),
                 name: abi_struct_entry.name,
                 size: abi_struct_entry.size,
-                members: abi_struct_entry.members.into_iter().map(|v| AbiStructMemberWrapper::from(v)).collect(),
+                members: abi_struct_entry.members.into_iter().map(AbiStructMemberWrapper::from).collect(),
             }
         }
     }
 
-    impl Into<LegacyStructAbiEntry> for AbiStructEntryWrapper {
-        fn into(self) -> LegacyStructAbiEntry {
+    impl From<AbiStructEntryWrapper> for LegacyStructAbiEntry {
+        fn from(abi_struct_entry: AbiStructEntryWrapper) -> Self {
             LegacyStructAbiEntry {
-                r#type: self.r#type.into(),
-                name: self.name,
-                size: self.size,
-                members: self.members.into_iter().map(|v| v.into()).collect(),
+                r#type: abi_struct_entry.r#type.into(),
+                name: abi_struct_entry.name,
+                size: abi_struct_entry.size,
+                members: abi_struct_entry.members.into_iter().map(|v| v.into()).collect(),
```

Comment:
Same thing here, replace calls to `v.into()`

Reply:
use `from` implemention

Reply:
not possible